playbooks/vars/main.yml

```diff
@@ -31,4 +31,17 @@ octavia_client_cert: "{{ cert_dir }}/client.pem"
 
 neutron_octavia_request_poll_timeout: "{{ '1000' if lookup('env', 'DEPLOY_AIO') == 'yes' else '100' }}"
 octavia_git_install_commit: d9e24e83bbf7d12f8bec16072b28c6ca655690ac #Head of Octavia stable/pike as of 4.1.2018
-requirements_git_install_branch: 99d99fe2e22f4a464415eb0064313b6ebd36906f #HEAD of "stable/pike" as of 4.10.2017
\ No newline at end of file
+requirements_git_install_branch: 99d99fe2e22f4a464415eb0064313b6ebd36906f #HEAD of "stable/pike" as of 4.10.2017
+
+octavia_num_cores: 1000
+octavia_num_instances: 1000
```

Comment:
So this means 1000 standalone load balancers or 500 Active/Standby load balancers.
This seems a bit low for a typical cloud.
Why not 10,000?

Reply:
done